{
	"id": "SUSE-SU-2024:2616-1",
	"modified": "2024-07-29T10:33:46Z",
	"published": "2024-07-29T10:33:46Z",
	"summary": "Security update for python-Django",
	"details": "This update for python-Django fixes the following issues:\n\n- Fixed regression in previous CVE fixes that made the package not work with Python 3.6\n",
	"affected": [
		{
			"package": {
				"ecosystem": "openSUSE:Leap 15.5",
				"name": "python-Django",
				"purl": "purl:rpm/suse/python-Django&distro=openSUSE%20Leap%2015.5"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "2.0.7-150000.1.23.1"
						}
					]
				}
			],
			"ecosystem_specific": {
				"binaries": [
					{
						"python3-Django": "2.0.7-150000.1.23.1"
					}
				]
			}
		}
	],
	"references": [
		{
			"type": "ADVISORY",
			"url": "https://www.suse.com/support/update/announcement/2024/suse-su-20242616-1/"
		}
	]
}
